src/components/uielements/input/InputTextArea.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
```

Comment:
Same here: Why do we need this `InputTextArea` component? It just wrapped `InputTextAreaWrapper`. A better approach would be just to define `InputTextArea` in `Input.style` and any other component can import it if needed.

Reply:
I initially did it that way. The problem is that the Input component extends InputProps and the Input.TextArea component extends TextAreaProps. It breaks when trying to spread InputProps over the Input.TextArea component. 

![image](https://user-images.githubusercontent.com/5379790/86572248-f53fd680-bf72-11ea-88d1-a5a9bdb2e359.png)

Reply:
@veado im not sure that this is a better idea. in this case there is gonna be a 'monster' component after some time which should resolve what it should use with a lot of `if` conditions. i believe it's gonna be really hard maintainable. antd decided the same way and splitted it to separate components with providing static methods:
![image](https://user-images.githubusercontent.com/67793194/86762222-2beb1f00-c04f-11ea-843a-9c64df81ffa3.png)